crashsimilarity/downloader.py

```diff
@@ -40,19 +54,40 @@ def download_signatures(self, bug_id):
 
         params = {'id': bug_id}
         response = self.get_with_retries(self._URL, params)
-        signatures = set()
-        for sig in self._json_or_raise(response)['bugs'][0]['cf_crash_signature'].split('\r\n'):
-            pos = sig.find('[@')
-            if pos != -1:
-                sig = sig[pos + 2:]
-            pos = sig.rfind(']')
-            if pos != -1:
-                sig = sig[:pos]
-            signatures.add(sig.strip())
+        signatures = self._json_or_raise(response)['bugs'][0]['cf_crash_signature']
+        cleaned_signatures = self._clean_signatures(signatures)
 
         if self._cache:
-            self._cache[key] = list(signatures)
-        return list(signatures)
+            self._cache[key] = cleaned_signatures
+        return cleaned_signatures
+
+    def download_bugs(self, from_date, to_date):
+        """
+        :param from_date: string "YYYY-MM-DD"
+        :param to_date: string "YYYY-MM-DD"
+        :return: list of bugs
+        """
+        params = {'include_fields': 'id,cf_crash_signature',
+                  'chfield': '[Bug creation]',
+                  'chfieldfrom': from_date,
+                  'chfieldto': to_date,
+                  'f2': 'cf_crash_signature',
+                  'o2': 'isnotempty',
+                  'product': ['Firefox', 'Core']}
+
+        key = ('bugzilla_bugs', json.dumps(params), utils.utc_today())
```

Comment:
Let's remove the cache as it's premature optimization. We can add it in the future if needed.
I expect the code that is going to use this module is going to store the downloaded bugs on disk to avoid requesting the same information from Bugzilla all the time, so probably the cache won't be needed.

Reply:
I agree, should I only fetch bugs with multiple signatures? as bugs with single signature won't help us in testing the model?

Reply:
Let's fetch them all. In the future we might have additional ideas on how to test the model that don't require multiple signatures, so it's good to have all the data handy (hopefully it won't be a HUGE amount of data, so it should be feasible).